src/x86_64/bindings.rs

```diff
@@ -113,7 +86,7 @@ where
     }
 }
 #[repr(C)]
-#[derive(Default)]
+#[derive(Default, Serialize, Deserialize)]
```

Comment:
Mh, I feel like implementing `Serialize` and `Deserialize` for this (and structs that contain it) is wrong, because it does not actually properly (de)serialize the FAM. Instead (de)serialization has to go through the `FamStructWrapper`. I guess for the `kvm_xsave` we can hand-write the (wrong) (De)Serialize implementations, until we properly support `KVM_CAP_XSAVE2`.

On the other hand, we already had (wrong) Versionize implementations here, so if you just want to maintain the status quo, that's also fine with me (although thinking longterm, these are something that shouldn't live upstream, so we could just use the opportunity to remove them now)

Reply:
I was wrong, the `Serialize` implementation for `FamStructWrapper` requires these types to have `Serialize` derived (same for `Deserialize`). This comment was wrong